cypress/e2e/27-two-factor-authentication.cy.ts

```diff
@@ -37,7 +37,7 @@ describe('Two-factor authentication', () => {
 		});
 	});
 
-	it('Should be able to login with MFA token', () => {
+	it.skip('Should be able to login with MFA token', () => {
```

Comment:
can we try to fix them instead of disabling?

Reply:
The description has the followup story for Ricardo, already discussed with him.

Reply:
@RicardoE105 please follow up on this asap